### ISSUE TITLE
1.18.2 support

### ISSUE THREAD

**Issue by Folas1337** [OskarsMC-Network/StaffPass#1]
1.18.2 is no longer supported :(

Error message:
```[11:17:41 INFO]: [StaffPass] Enabling StaffPass v1.0
[11:17:41 WARN]: [StaffPass] Loaded class us.myles.ViaVersion.api.protocol.ProtocolRegistry from ViaVersion v4.2.1 which is not a depend or softdepend of this plugin.
[11:17:41 ERROR]: Error occurred while enabling StaffPass v1.0 (Is it up to date?)
java.lang.NoSuchMethodError: 'net.kyori.adventure.text.minimessage.MiniMessage net.kyori.adventure.text.minimessage.MiniMessage.get()'
        at com.oskarsmc.staffpass.StaffPass.onEnable(StaffPass.java:33) ~[staffpass.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:501) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugin(CraftServer.java:559) ~[paper-1.18.2.jar:git-Paper-231]
        at org.bukkit.craftbukkit.v1_18_R2.CraftServer.enablePlugins(CraftServer.java:473) ~[paper-1.18.2.jar:git-Paper-231]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:666) ~[paper-1.18.2.jar:git-Paper-231]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:433) ~[paper-1.18.2.jar:git-Paper-231]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:316) ~[paper-1.18.2.jar:git-Paper-231]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1165) ~[paper-1.18.2.jar:git-Paper-231]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.2.jar:git-Paper-231]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
[11:17:41 INFO]: [StaffPass] Disabling StaffPass v1.0```

**Comment by OskarZyg** [comment]
Thanks, I'll look into it, paper 1.18.2 now shades mini message 4.10.0, while this plugin is using an ancient version of it.

**Comment by OskarZyg** [comment]
Fixed with release 1.0.0
